graphene_django_extras/utils.py

```diff
@@ -314,6 +307,7 @@ def recursive_params(
 
         if isinstance(field, InlineFragment):
             a, b = recursive_params(
+                root_info,
```

Comment:
You added two parameters to this function definition, but you're only passing one more here.